qemu_v8.mk

```diff
@@ -141,6 +195,37 @@ edk2: edk2-common
 
 edk2-clean: edk2-clean-common
 
+################################################################################
```

Comment:
This is perhaps something that could be moved to `common.mk` instead. There's quite a few platforms now compiling U-Boot using a somewhat similar pattern, probably due to copy paste + some small tweak in each case.

Reply:
Ok, will do